src/local/butler/reproduce.py

```diff
@@ -423,6 +399,12 @@ def execute(args):
   # Initialize fuzzing engines.
   init.run()
 
+  # Prepare the emulator if needed.
+  emulator_process = None
+  if args.emulator:
+    print('Starting emulator...')
+    emulator_process = android.start_emulator()
```

Comment:
We can fail in start_emulator if it is not installed and provide instuction to people to install it.

Reply:
With reproduce.sh this shouldn't happen since we'll detect it and install it automatically. There's some argument for providing instructions in every case if people try to use butler.py directly, but I don't want to special-case this one here.